.not-formatted

```diff
@@ -42,47 +42,19 @@
 ./pdns/devpollmplexer.cc
```

Comment:
Hmm, sort -c agrees here on the old order, using the C locale. How did you sort this file? It's hard to see what's the actual changes are right now.

Reply:
Good catch! Wasn't using the C locale.